src/Components/Link/index.jsx

```diff
@@ -0,0 +1,15 @@
+
+import PropTypes from 'prop-types'
+function Link ({ title }) {
```

Comment:
Please change with arrow function.
*based on pattern

Reply:
thankyou for your review, this issue fix in 327bdc89e17135a0db052aba8aa4120c69ceb0f2